config/projects.yaml

```
@@ -364,14 +364,14 @@ media-bundle:
     branches:
         master:
             php: ['7.3', '7.4']
-            services: [mongodb]
+            services: [mongodb, soap, gd]
```

```suggestion
            services: [mongodb]
            php-extensions: [soap, gd]            
```

plus config change, adjust the value objects and read and apply them to the CI templates